api/filter/http/squash.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+// [#proto-status: experimental]
+
+package envoy.api.v2.filter.http;
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/struct.proto";
+import "validate/validate.proto";
+
+// [#protodoc-title: Squash]
+// Squash :ref:`configuration overview <config_http_filters_squash>`.
+
```

Comment:
Actually, I think the issue is here. Delete his newline, and a newline before the message. Basically, all the docs need to be top level and attached.

Reply:
+1, that will be the issue.